src/commands/PublishCommand.js

```diff
@@ -151,6 +151,20 @@ export default class PublishCommand extends Command {
 
   getVersionsForUpdates(callback) {
     if (this.flags.repoVersion) {
+      // Allows automatic bumping to next semver via repoVersion flag
+      if (this.flags.repoVersion === "patch" ||
+          this.flags.repoVersion === "minor" ||
+          this.flags.repoVersion === "major" ||
+          this.flags.repoVersion === "current"
+      ) {
+        const versions = {};
+        this.updates.forEach((update) => {
+          versions[update.package.name] = this.flags.repoVersion === "current" ?
+            update.package.version : semver.inc(update.package.version, this.flags.repoVersion);
+        });
+
+        return callback(null, { versions });
+      }
```

Comment:
I had the same thoughts :) This was the easiest point of interception to get this working locally without introducing much new functionality. Stoked maintainers are willing to help guide this one, we really need it. Any suggestions on independent flag name? I feel it should be fairly straight forward within the same method to intercept and alter the test. I'll get back to you with more suggestions on that first thing next week. As I stated in the comment, definitely tough to get all paths (especially the inverse coverage) when you have many options